LDRH_choice.py

```diff
@@ -460,16 +491,28 @@ def run_staircase(task, operation=None):
     except StopIteration:
         output={}
         output['score'] = 'StopIteration'
-        output['thisIncrement'] = handler.intensities[-1]
+        if not use_posterior_matching:
+            output['thisIncrement'] = handler.intensities[-1]
 
     return output
 
+if testing_questionnaire:
```

Comment:
it's because I'm currently using it for testing, so I can test right at the beginning of the task. Since we need this for camp tomorrow, I don't want to do the work over and over of copying the code and removing it again. So, I just added 'testing_questionnaire' that can be set as true or false for the purpose of testing.